harold/plugins/deploy.py

```diff
@@ -41,12 +42,28 @@ class DeployConfig(PluginConfig):
     blackout_hours_end = Option(parse_time)
 
 
+class DeployHoldType(Enum):
```

Comment:
My thought here is that a code freeze is the same thins as a hold.  For code freezes we can leverage the existing hold logic already built into Harold.

Reply:
agreed